dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -354,6 +356,45 @@ InsertItemToListView(
     return -1;
 }
 
+static BOOL tmToStr(
+    IN struct tm *pTM,
+    OUT LPWSTR szBuffer,
+    IN UINT nBufferSize)
+{
+    SYSTEMTIME st;
+    LPWSTR szBufferDate;
+    LPWSTR szBufferTime;
+    UINT nCharDate, nCharTime;
+    BOOL bResult;
+
+    bResult = FALSE;
+    st.wYear = pTM->tm_year + 1900;
+    st.wMonth = pTM->tm_mon + 1;
+    st.wDay = pTM->tm_mday;
+    st.wHour = pTM->tm_hour;
+    st.wMinute = pTM->tm_min;
+    st.wSecond = pTM->tm_sec;            
+    
+    //Check required size before cpy/cat
+    nCharDate = GetDateFormatW(LOCALE_USER_DEFAULT, 0, &st, nullptr, NULL, 0);
+    nCharTime = GetTimeFormatW(LOCALE_USER_DEFAULT, 0, &st, nullptr, NULL, 0);
```

Comment:
```suggestion
    nCharDate = GetDateFormatW(LOCALE_USER_DEFAULT, 0, &st, NULL, NULL, 0);
    nCharTime = GetTimeFormatW(LOCALE_USER_DEFAULT, 0, &st, NULL, NULL, 0);
```

Reply:
Here I changed back to NULL because, in this module it appears we just always use NULL, and also in the same function call you were using a mixture of nullptr and NULL.

Reply:
`nullptr` comes from language support in the compiler,
`NULL` is from our own sdk

Reply:
OK, NULL to be used.